doc/guide/src/develop-in-docker-container.md

```diff
@@ -30,6 +30,7 @@ docker-compose exec dev bash
 
 After that, run the setup script:
 ```
+chmod +x ./setup.sh
```

Comment:
Git can track the permissions for a file, so I think we can just modify it with executable permissions and commit it to the repo to reduce the steps for local development.

Reply:
As I understand it, while Git can track file permissions, there are some cases where developers may not be able to run setup.sh in a Docker container due to the nature of Docker's bind mount system.

Additionally, I have personally encountered situations where permission was explicitly required within the container to execute the script.